andela/strikers-ah-frontend
# Ft crop image 166561722

#### What does this PR do?
- This pull request creates a crop image functionality to the profile page before upload
#### Description of Task to be completed?
- reset image orientation and preview it
- crop image
#### How should this be manually tested?
- login ```/login``` to access the profile page ```/profile/: username``` and then try to upload an image you will see a modal which will help you to crop that image  
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/166561722
#### Screenshots (if appropriate).
- ![image](https://user-images.githubusercontent.com/36665761/59285606-865c5500-8c6e-11e9-81ab-d003d72d24a2.png)
